File: Python/Task16/n27413.py
# https://inf-ege.sdamgia.ru/problem?id=27413

def F(n):
	if n == 1:
		return 1
	elif n % 2 == 0:
		return n + F(n - 1)
	else: # if n % 2 == 1
		return 2 * F(n - 2)

print(F(26))

File: C++/Task16/n27413/n27413.cpp
// https://inf.reshuege.ru/problem?id=27413

#include <iostream>

int F(int n)
{
	if (n == 1)
	{
		return 1;
	}
	else if (n % 2 == 0)
	{
		return n + F(n - 1);
	}
	else // if(n % 2 == 1)
	{
		return 2 * F(n - 2);
	}
}

int main()
{
	int r = F(26);
	std::cout << r << std::endl;
	return 0;
}